test/data/minimal_example.in.txt

```diff
@@ -1,4 +1,3 @@
 Chromosome	Start_Position	End_Position	Reference_Allele	Tumor_Seq_Allele1
 3	178916927	178916939	TAGGCAACCGTGA	G
 7	55220240	55220240	G	T
-7	55242467	55242486	AATTAAGAGAAGCAACATCT	AGCAA
```

Comment:
7,55242467,55242486,AATTAAGAGAAGCAACATCT,AGCAA is failing because it is normalized to 7,55242468,55242486,ATTAAGAGAAGCAACATCT,GCAA. It couldn't pass our validator because input_reference_allele does not equal to annotation_reference_allele. We should do: input -> normalization -> annotate normalized input with vep -> validate normalized result